### ISSUE TITLE
When going through lists, every time you disable an album, the list resets back to the first page

### ISSUE THREAD

**Issue by zaheria985** [lejacobroy/Tidalrr#16]
When going through lists, every time you disable an album, the list resets back to the first page.  It's a little annoying when you are trying to weed out stuff you don't want and you've got a lot to go through.

**Comment by zaheria985** [comment]
It also resets any sorts you've done and resets your total albums per page back to 10.  Pretty minor issues I guess, just driving me a bit nuts.  Thanks for the hard work you've put into this though!

**Comment by lejacobroy** [comment]
Yes, the table and filtering is provided by in-browser js and resets on page refresh.
To fix this, I would need to implement paging in the flask routes and jinja2 templates, which are not planned anytime soon, sorry!
